src/compiler/checker.ts

```diff
@@ -7278,6 +7291,9 @@ export function createTypeChecker(host: TypeCheckerHost): TypeChecker {
                 for (const type of types) {
                     i++;
                     if (checkTruncationLength(context) && (i + 2 < types.length - 1)) {
+                        if (context.flags & NodeBuilderFlags.NoTruncation) {

```

Comment:
Am I confused or is this truncating things _more_ when the flag is enabled?

Reply:
It is not more per se, but it is replacing the previous values with an `any`

We kinda have 3 states: when `NoTruncation` is disabled, we truncate up to `defaultMaximumTruncationLength`, if `NoTruncation` is enabled it truncates up to `noTruncationMaximumTruncationLength`, anything after that is an `any`.

Reply:
If we decide to remove all of the types and just have an `any` it will not make sense to have this message as the code would like like:
```
 foo(bar: /*... 516 more elided ...*/ any): /*elided*/ any {
        throw new Error("Method not implemented.");
    }
    get bar(): /*... 516 more elided ...*/ any {
        throw new Error("Method not implemented.");
    }
    set bar(value: /*... 516 more elided ...*/ any) {
        throw new Error("Method not implemented.");
    }
    baz<V extends /*... 516 more elided ...*/ any>(value: V): V {
        throw new Error("Method not implemented.");
    }
```

We can instead count all of the types that were elided and put that:
```
 foo(bar: /*... 800 elided ...*/ any): /*... 800 elided ...*/ any {
       throw new Error("Method not implemented.");
 }
```